doc/modules/recommended_practices.rst

```diff
@@ -0,0 +1,30 @@
+=====================
+Recommended Practices
+=====================
+
+
+Feature Selection
+=================
+
+**Wrong Practice: Selecting features using Whole Dataset**
+
+    >>> X = numpy.random.randn(500,10000)
+    >>> y = numpy.random.choice(2,500)
```

Comment:
To make the code snippet easier to understand for people not familiar with the meaning of the argument of numpy random functions I would rather do:

```python
    >>> n_samples, n_features, n_classes = 500, 10000, 2
    >>> X = numpy.random.randn(n_samples, n_features)
    >>> y = numpy.random.choice(n_classes, n_samples)
```

Reply:
One could also reduce the number of samples to 200 or so to make the effect even more dramatic.